precompile/wasm/IWasm.sol

```diff
@@ -32,20 +32,21 @@ interface IWasm {
     function executeContract(
         address sender,
         address contractAddress,
-        bytes calldata msg,
+        bytes calldata message,
         string calldata denom,
         uint256 amount
     ) external returns (bytes calldata data);
     function migrateContract(
         address sender,
         address contractAddress,
         uint256 codeId,
-        bytes calldata msg
+        bytes calldata message
```

Comment:
Is there a reason why msg was changed to message? wasmd's json type currently uses msg.

Reply:
`msg` is reserved variable for tx information in solidity (e.g. tx sender -> `msg.sender`, time -> `msg.timestamp`)

Reply:
Rollbacked as [d5836fd](https://github.com/xpladev/xpla/pull/161/commits/d5836fdf2e82e087a89763de125eb7eb0e9d804f)